lua/orgmode/config/init.lua

```diff
@@ -200,21 +200,32 @@ function Config:setup_mappings(category)
     return
   end
 
-  for name, key in pairs(self.opts.mappings[category]) do
+  for name, lhs in pairs(self.opts.mappings[category]) do
     if mappings[category] and mappings[category][name] then
-      local map = vim.tbl_map(function(i)
-        return string.format('"%s"', i)
-      end, mappings[category][name])
-      local keys = key
-      if type(keys) == 'string' then
-        keys = { keys }
+      local map = {}
+      if #mappings[category][name] == 0 then -- we are in a multi-mode mapping
```

Comment:
I think we should use this check:
```lua
type(mappings[category][name]) == 'table' and not vim.tbl_islist(mappings[category][name]
```
Because we have situation where we have a mapping that is a table list, but is actually a single normal mapping with an additional parameter, like [this one](https://github.com/nvim-orgmode/orgmode/blob/master/lua/orgmode/config/mappings.lua#L78)

Reply:
If I'm not mistaken that example should work regardless, because it should still go into the else branch (since the length operator would return non-zero) :thinking:  But you are absolutely right, my check is ugly and I just didn't know how to do it properly :grin: Thanks a lot, will add the change :)